### PR TITLE
Properly write mobile ion species specified on form

### LINKED ISSUE #142: Mobile ion species not written to APBS input file
This issue pertains to a problem mentioned in Electrostatics/apbs-web#65

The `charge`, `conc`, and `radius` fields passed from the frontend form aren't being used when creating the resulting APBS input file.  Will need to verify that expected form fields are in use where appropriate.

### PR DESCRIPTION
**Pull request by Eo300**
Resolves #142 
Resolves Electrostatics/apbs-web#65

Added new test to cover this case.

### CODE REVIEW

**Review by intendo — APPROVED** [review]

File: tests/expected_data/1fas-ion.in [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	mol pqr 1fas.pqr
+end
+elec
+	mg-auto
+	dime 129 97 97
+	cglen 63.1108 45.73 61.4601
+	fglen 57.124 45.73 56.153
+	cgcent mol 1
+	fgcent mol 1
+	ion charge 1 conc 0.15 radius 0.95
```
Comment (sobolevnrm) [inline tests/expected_data/1fas-ion.in:11]: APBS can't distinguish between ions with different radii. Can we please use 2.0 for both ion radii?

File: tests/expected_data/1fas-ion.in [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	ion charge -1 conc 0.15 radius 1.81
```
Comment (sobolevnrm) [inline tests/expected_data/1fas-ion.in:12]: APBS can't distinguish between ions with different radii. Can we please use 2.0 for both ion radii?

File: tests/input_data/test_apbs_setup-1fas-ion.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    "bcfl": "sdh",
+    "chgm": "spl2",
+    "srfm": "smol",
+    "charge0": "1",
+    "charge1": "-1",
+    "charge2": "",
+    "conc0": "0.15",
+    "conc1": "0.15",
+    "conc2": "",
+    "radius0": "0.95",
```
Comment (sobolevnrm) [inline tests/input_data/test_apbs_setup-1fas-ion.json:24]: APBS can't distinguish between ions with different radii. Can we please use 2.0 for both ion radii?

File: tests/input_data/test_apbs_setup-1fas-ion.json [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    "radius1": "1.81",
```
Comment (sobolevnrm) [inline tests/input_data/test_apbs_setup-1fas-ion.json:25]: APBS can't distinguish between ions with different radii. Can we please use 2.0 for both ion radii?

**Review by sobolevnrm — CHANGES_REQUESTED** [review]
Thanks for the quick fix!  I have some minor comments.

File: tests/expected_data/1fas-ion.in (hunk 1 above)
Comment (Eo300) [reply tests/expected_data/1fas-ion.in]: The test file's been updated to use `2.0` for this radius

File: tests/input_data/test_apbs_setup-1fas-ion.json (hunk 3 above)
Comment (Eo300) [reply tests/input_data/test_apbs_setup-1fas-ion.json]: The test file's been updated to use `2.0` for this radius

File: tests/input_data/test_apbs_setup-1fas-ion.json (hunk 4 above)
Comment (Eo300) [reply tests/input_data/test_apbs_setup-1fas-ion.json]: The test file's been updated to use `2.0` for this radius

File: tests/expected_data/1fas-ion.in (hunk 2 above)
Comment (Eo300) [reply tests/expected_data/1fas-ion.in]: The test file's been updated to use `2.0` for this radius

**Review by sobolevnrm — APPROVED** [review]

**Review by intendo — APPROVED** [review]

**Pull request merged**